doc/api/worker_threads.md

```diff
@@ -61,6 +61,26 @@ Worker threads inherit non-process-specific options by default. Refer to
 [`Worker constructor options`][] to know how to customize worker thread options,
 specifically `argv` and `execArgv` options.
 
+## `worker.connect(target[, data][, timeout])`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1.1 - Active development
+
+* `target` {number} The target thread id.
+* `data` {any} Any arbitrary, cloneable JavaScript value.
```

Comment:
Would be good to indicate in here what the `data` is used for in this case. Also, what about transferable values / transfer list?

Reply:
Added!

Reply:
So far I explicitly don't support transferrable as the connect callback is only meant to send small data to handshake the connection.
If the thread needs to transfer objects, it can do that later once the communication is established.
Do you think we explicitly need transferable here?

Reply:
> Do you think we explicitly need transferable here?

It depends on the case, I think. Some objects might *only* support transfer here. For instance, this limitation would mean that it's impossible to use a `ReadableStream` as the `data`... which might be perfectly acceptable for the use case but the limitation should be explicitly called out and explained in the docs.

Reply:
I added a note, does it look fine now?